src/main/kotlin/kr/io/snuhbmilab/carediaryserverv2/domain/user/facade/UserFacade.kt

```
@@ -56,9 +66,20 @@ class UserFacade(
         return UserRegisterResponse.from(accessToken)
     }
 
+    private fun validateVerificationCode(role: Role, adminCode: String?) {
+        val expectedCode = when (role) {
+            Role.ADMIN -> verificationCodeProperties.admin
+            Role.CARE_MANAGER -> verificationCodeProperties.careManager
+            else -> throw IllegalStateException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)